docs/_docs/manager.md

```diff
@@ -124,15 +132,16 @@ Data obtained through questionnaires
 
 
 ### Data export
+(**Partially implemented**)
 
-(**Not yet implemented**)
-
-The customer should be able to export data in CSV format to reuse them in mass marketing software (eg. MailChimp).
+The customer is able to export data in CSV format to reuse them in mass marketing software (eg. MailChimp).
 
 Exportable data should include:
 
-* Profile users according to the characteristics
-* Export emails only for specific campaigns
-* Subset of standardized profiles (?)
+* email 
+* phone number
+* username 
+
+Other data (like gender, age...) we can receive from social login are still not exportable,the system will export ONLY data from users that allow marketing usage.
```

Comment:
I'd change to something like this:

Some extra data from social logins, like gender and age,  are still not exportable.
Also note that the system will export data only for users who explicitly gave marketing consensus during login.